routes/post.go

```diff
@@ -837,29 +871,29 @@ func (fes *APIServer) GetPostsStateless(ww http.ResponseWriter, req *http.Reques
 			profileEntryMap,
 			readerStateMap,
 			err = fes.GetPostEntriesForFollowFeed(startPostHash, readerPublicKeyBytes, numToFetch, utxoView,
-			requestData.MediaRequired)
+			requestData.MediaRequired, requestData.OnlyNFTs, requestData.OnlyPosts)
```

Comment:
validate these values earlier in this function please. should be right after we parse the request since comparing two bools is cheaper than most other things we do validation wise.